content/en/docs/user-documentation/networking/mdns.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "mDNS"
+weight: 30
+description: embedded Multicast DNS support in microshift.
+card:
+  name: networking
+  weight: 30
+---
+
+MicroShift includes an embedded mDNS server. mDNS is a protocol used to allow
```

Comment:
Maybe directly in or after the first sentence explain why/when this is useful, e.g. "... for deployment scenarios in which the authoritative DNS server cannot be reconfigured to point clients to services on MicroShift.".

Then new paragraph "mDNS is..."

Reply:
+1